dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessInstanceServiceImpl.java

```diff
@@ -335,6 +339,19 @@ public Result queryProcessInstanceList(User loginUser, long projectCode, long pr
         return result;
     }
 
+    /**
```

Comment:
Since this method is not an open API, and its logic is quite clear, I prefer removing the comments. As u could see, there are many legacy redundant comments in this project which are hard to maintain, we will remove and fix those comments in the future gradually.

Reply:
sure, remove it